aitelemetry/api.go

```diff
@@ -14,6 +14,13 @@ type Report struct {
 	CustomDimensions map[string]string
 }
 
+// Application event structure
+type AiEvent struct {
```

Comment:
Can be named as Event to be consistent with report and metric

Reply:
Renamed.